tests/PriorityBoosting.test.js

```diff
@@ -0,0 +1,114 @@
+const sinon = require('sinon');
```

Comment:
Does this test work? It would be awesome to incorporate it into the actual sprint for students to test their stretch goal code on.

Reply:
All tests including PriorityBoosting are passing.   I would like to make some test improvements.